frontend/e-buddy/src/app/(demo)/bots/viewbots/page.tsx

```
@@ -162,58 +165,59 @@ export default function MyBotsPage() {
                         className="w-10 h-10 rounded-full flex items-center justify-center"
                         style={{
                           backgroundColor:
-                            (bot.customization?.primaryColor || "#6366f1") + "20"
+                            (bot.customization?.primaryColor ?? "#6366f1") + "20"
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)